tests/test_convergence.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+# Author: Austin Zadoks <austin.zadoks@epfl.ch>
+"""
+Tests for the OptimizationWorkChain.
+"""
+
+import pytest
+
+from aiida_optimize.engines import Convergence
+
+
+@pytest.fixture
```

Comment:
Not sure if these need to be a fixture, since they're used only once. But I guess the idea is that they'd be reused when adding more tests?

Total side note, but I really like `pytest-cases` for writing a single test and then throwing different scenarios at it.

Reply:
I removed the fixtures and hard-coded the inputs to the tests